great_expectations/expectations/core/expect_column_values_to_not_be_in_set.py

```diff
@@ -143,16 +142,16 @@ def _prescriptive_template(
         cls,
         renderer_configuration: RendererConfiguration,
     ) -> RendererConfiguration:
-        add_param_args = (
+        add_param_args: List[AddParamArgs] = [
             ("column", RendererValueType.STRING),
             ("value_set", RendererValueType.ARRAY),
             ("mostly", RendererValueType.NUMBER),
             ("parse_strings_as_datetimes", RendererValueType.BOOLEAN),
-        )
+        ]
```

Comment:
All things being equal, I think. a tuple would be better since it's immutable.

Reply:
I think I made this change because I forgot you can annotate a tuple of unknown length `Tuple[AddParamArgs, ...]`. I will make this change in the next PR since I need to change some of it as well.